Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -173,6 +173,10 @@ void RenderBox::willBeDestroyed()
             view().unregisterAnchor(*this);
         if (!style().positionTryFallbacks().isEmpty())
             view().unregisterPositionTryBox(*this);
+        if (isOutOfFlowPositioned()) { // Keep this in sync with styleDidChange().
+            if (Style::AnchorPositionEvaluator::isLayoutTimeAnchorPositioned(style()) || style().usesAnchorFunctions())
```

Comment:
These are equivalent to `if (Style::AnchorPositionEvaluator::isAnchorPositioned(style()))`